next.clayui.com/src/utils/getPackageConfig.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const {readFileSync} = require('fs');
+const {join, resolve} = require('path');
+
+function getPackageConfig(name) {
```

Comment:
hey @diegonvs this may break with a recent change in master, there is the case of Popover and Tooltip that `packageNpm` has two packages, so as not to break this we can separate the documentation.

Reply:
Nice, I broke `Popover and Tooltip` into separated ones.